appveyor.yml

```diff
@@ -1,34 +1,44 @@
-# Fix line endings on Windows
-init:
-  - git config --global core.autocrlf true
```

Comment:
It shouldn't be needed, though. If tests fail because of that, we should just normalize the line endings.